### PR TITLE
Added ability to add comments

### PR DESCRIPTION
**Pull request by constantine-prokopenko-vrp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,7 @@ See also [Uploading a SARIF file to GitHub](https://docs.github.com/en/code-secu
...
 |`createGitHubAnnotations`|no|"true"|By default, all detected violations are added as annotations to the pull request. You can disable this by setting FALSE. This can be useful if you are using another tool for this purpose.|
+|`createGitHubComments`|no|"false"|This flag used when you want to see violations as comments with the following format: `{priorityIcon} ({priority}) - {fileName} (Line:{startLine})`|
```
Comment (adangel) [inline README.md:86]: This new property also needs to be added in `action.yml`, so that it is actually usable.

**Review by adangel — COMMENTED** [review]
Thanks for the PR.
I hope I find time the next days to test it (e.g. with a manual integration test -> https://github.com/pmd/pmd-github-action-tests )

File: README.md (hunk 1 above)
Comment (constantine-prokopenko-vrp) [reply README.md:86]: Many thanks for your comment @adangel.
The `createGitHubComments` property added to `action.yml`.

**Comment by adangel** [comment]
I tested this now, but unfortunately no comments are created for me. Did you have any success?

I don't see any output from octokit, so I'm not even sure, whether the POST request is executed at all.

Possible problems:
- creating issue comments requires write permission. A PR from a forked repository only has read permissions (see https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions ).
- The default token, that is generated for a action, might not necessarily work for that reason.
- Since we use the token now also to create comments, we should probably update the documentation for the `token` parameter (it's now not only used for "analyzeModifiedFilesOnly").
- Executing the POST Requests locally with curl with my own PAT works. (Note: I added some logging in https://github.com/pmd/pmd-github-action/commit/d80fd9479bd638accbc94674bf4449d6ddfe2e46 and https://github.com/pmd/pmd-github-action/commit/efcf6fa3ec3e8a865d110597a3953398d03b1021). So, the general data should be fine, but since Octokit doesn't log anything at all...
- I tried around a bit with await/async, since the request is a promise and is executed asynchrounously, so it might happen, that the request are not executed before the action finishes. But that didn't change anything (https://github.com/pmd/pmd-github-action/commit/60d1c0e3f038880f52797261e54d2bbfe96e5a09, https://github.com/pmd/pmd-github-action/commit/29996a59f7d5d6ad33c4c379a62335fabe9ec296, https://github.com/pmd/pmd-github-action/commit/1462bb4518b1ed23795561cad4877fe25d7bb4f4)
- See https://github.com/pmd/pmd-github-action-tests/pull/15 for the last Test PR (which is from the same repo, so it indeed has write access to create comments).

**Comment by constantine-prokopenko-vrp** [comment]
Many thanks for your investigation. I'll close the PR for now. I'll create a new PR once I resolve all the issues.

**Pull request closed without merging**